### PR TITLE
create home page

### PR DESCRIPTION
**Pull request by git-0r**
# Home page for e-commerce
ecom-home-1
ecom-home-2
ecom-home-3

Please review 'src' folder. Number of file changes is high because it's first PR of this project. Thank you.

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { useEffect } from "react";
+import { getProducts } from "../api-calls";
+import Banner from "../components/Banner";
+import Footer from "../components/Footer";
+import Header from "../components/Header";
+import IntroVideo from "../components/IntroVideo";
+import Navbar from "../components/Navbar";
+import NewArrivals from "../components/NewArrivals";
+import PopularCategory from "../components/PopularCategory";
```
Comment (yaswanthmyneni) [inline src/pages/Home.js:9]: suggestion: add index.js in components folder and export all components from it

**Review by yaswanthmyneni — APPROVED** [review]
The UI looks great, you can merge.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            </div>
+            <div className="nav-section-center d-flex flex-justify-evenly">
+                <p className="d-flex flex-justify-evenly flex-align-center">Shop
+                    <ion-icon name="caret-down-outline"></ion-icon>
+                </p>
+                <p className="d-flex flex-justify-evenly flex-align-center">Our wines
+                    <ion-icon name="caret-down-outline"></ion-icon>
+                </p>
+                <p>About us</p>
+                <p>Blog</p>
```
Comment (Naman-Saxena1) [inline src/components/Navbar.js]: I think for semantic HTML, it would be better to have them as buttons rather than p tag.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks fine.
I have 1 UI suggestion
Capture3

I believe bottle overflowing is intentional and that is fine.
But text is overflowing differently for different cards and looks strange for consistent UI design, so can do some alignment on it.

Everything else looks great 👍

**Review by Naman-Saxena1 — APPROVED** [review]

**Pull request merged**